src/config/getDomainSpecificContent.js

```diff
@@ -0,0 +1,33 @@
+const { ACCOUNT_ID } = require('../api.v2/constants');
+
+const domainSpecificContent = {
+  test: {
+    githubOrganisationName: 'org',
+    adminEmail: 'admin@example.com',
+    enforcePrivacyPolicyAgreement: true,
```

Comment:
As it is, local development is broken for me (I log in to HMS staging with no privacy policy)

```suggestion
    enforcePrivacyPolicyAgreement: false,
```

Reply:
This is for the test environment, did you run the api with `DEV_ACCOUNT=HMS npm start` ?

Reply:
`DEV_ACCOUNT` is not used by the `api` AFAIK.

Look down at `getDomainSpecificContent`: it checks for `AWS_ACCOUNT_ID` which for development needs to be unset or `'000000000000'` (the default that is used if unset -- see `default-config.js`). As a result, `getDomainSpecificContent` returns `domainSpecificContent.test` and I can't develop locally due to complaints from the API about not agreeing to privacy policy.

`DEV_ACCOUNT=HMS npm start` is run in the ui. I tried also doing in the api but nothing different (also as mentioned `DEV_ACCOUNT` isn't anywhere in the api code base). 

I also tried setting `AWS_ACCOUNT_ID` to HMS appropriate value, which fixes the privacy policy issue but creates local infrastructure communication issues (cant upload data) because the localhost aws account needs to be `'000000000000'`.